benchmark/src/main/java/ai/timefold/solver/benchmark/impl/report/LineChart.java

```diff
@@ -209,25 +223,35 @@ public LineChart<X, Y> build(String fileName, String title, String xLabel, Strin
              * This allows Chart.js to only draw the absolute minimum necessary points.
              */
             data.values().forEach(map -> {
-                List<Map.Entry<X, Y>> entries = map.entrySet().stream().toList();
+                List<Entry<X, Y>> entries = map.entrySet().stream().toList();
```

Comment:
I'd really like us to start using `var` for these generic-heavy things.
The declaration brings no additional context here. You already know it's a list (`toList()`) and you already know it is an entry (`entrySet()`).